storage/block.go

```diff
@@ -125,15 +135,12 @@ func (b *dbBlock) close() {
 	}
 
 	// Otherwise, we need to close the stream.
```

Comment:
Perhaps rename this to "Otherwise we need to return the bytes to the bytes pool".